fcrepo-stats-api/pom.xml

```diff
@@ -28,13 +28,13 @@
 
     <!-- test gear -->
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
+      <artifactId>mockito-junit-jupiter</artifactId>
```

Comment:
There are no tests in fcrepo-stats-api, should we just remove these dependencies?

Reply:
By "these" I mean `junit-jupiter` and `mockito-junit-jupiter`

Reply:
A little depressing, but sure why not, they are removed. Looks like all the classes are simple beans, only one does [slightly more than return a value](https://github.com/fcrepo/fcrepo/blob/main/fcrepo-stats-api/src/main/java/org/fcrepo/stats/api/RepositoryStatsByMimeTypeResults.java#L34).